index.html

```diff
@@ -221,7 +221,103 @@ <h2><i>Kiss And Tell</i></h2>
              
             </div><!--/.col-xs-6.col-lg-4-->
           </div><!--/row-->
-			              
+ 
+			                <br>
+					<br>
+					<h1 style = "font-size: 45px;text-align:center;" > Movies </h1>
+					
+					
+					
+						<div class="album col-xs-6 col-lg-4">
+						<h3>Neighbours 2</h3>
+						
+						<p><img src="img/neighbors2.jpeg " class="roundmovietiles" ></p>
```

Comment:
@thenullterminator  Instead of adding the movies again you just have to add the class="roundmovietiels" to existing movie section. Please fix it.